cds-snc/cppd-medical-report-sails
# Fix dashboard validation for documents

# What this does

Fixes the issue identified here: https://trello.com/c/o6v6TMEO/538-logic-for-supporting-documents-is-never-marked-as-complete-on-the-dashboard-page

## To test

- Make sure you've entered at least one condition
- Navigate to the Supporting Documents section
- You can either upload some files, or check the Attach Later checkbox
- The Supporting Documents section on the dashboard should show as complete
